pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: languagetool_textfield
 description: The LanguageTool TextField package is a spell-checker designed for Flutter apps. This is useful for apps that need text input like messaging, notes, and email.
-version: 0.0.6
+version: 0.0.7
```

Reprecussion in case there's some breaking changes from hiding some classes.
```suggestion
version: 0.1.0
```